src/metrics/HansonMetric.ts

```diff
@@ -0,0 +1,34 @@
+import { IMetric } from '@/types';
+import esprima from 'esprima';
+import McCabeCC from './McCabeCC';
+import NumberOfHansonOperators from './NumberOfHansonOperators';
+// @ts-ignore
+import * as Styx from 'styx';
+
+export default class HansonMetric implements IMetric {
+  private _name = 'Hanson metric';
+  private _info =
+    'Complexity metric defined by Hanson https://dl.acm.org/doi/pdf/10.1145/954373.954375';
+  private _scope = 'any';
```

Comment:
Are you sure that the scope is any?? How you gonna calculate cyclomatic complexity for the class or group of classes? Will it work?